packages/amplify-category-custom/resources/cdk-stack.ts.sample

```diff
@@ -1,10 +1,10 @@
-import * as cdk from '@aws-cdk/core';
+import * as cdk from 'aws-cdk-lib';
 import * as AmplifyHelpers from '@aws-amplify/cli-extensibility-helper';
 import { AmplifyDependentResourcesAttributes } from '../../types/amplify-dependent-resources-ref';
-//import * as iam from '@aws-cdk/aws-iam';
-//import * as sns from '@aws-cdk/aws-sns';
-//import * as subs from '@aws-cdk/aws-sns-subscriptions';
-//import * as sqs from '@aws-cdk/aws-sqs';
+//import * as iam from 'aws-cdk-lib/aws-iam';
+//import * as sns from 'aws-cdk-lib/aws-sns';
+//import * as subs from 'aws-cdk-lib/aws-sns-subscriptions';
+//import * as sqs from 'aws-cdk-lib/aws-sqs';
```

Comment:
why keep these in if they're commented out? lol

Reply:
That's part of experience. See full file https://github.com/aws-amplify/amplify-cli/blob/46ecbd0218144e80a0bcf916b4fa0832842cd9dc/packages/amplify-category-custom/resources/cdk-stack.ts.sample .